core/src/main/java/dev/streamx/cli/ingestion/BasePublishUnpublishCommand.java

```diff
@@ -0,0 +1,74 @@
+package dev.streamx.cli.ingestion;
+
+import dev.streamx.cli.exception.IngestionClientException;
+import dev.streamx.clients.ingestion.StreamxClient;
+import dev.streamx.clients.ingestion.exceptions.StreamxClientException;
+import jakarta.inject.Inject;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Spec;
+
+public abstract class BasePublishUnpublishCommand implements Runnable {
```

Comment:
There was many common code in both commands.
Now there is even more - so moving common code to this base class

Reply:
maybe name it BaseCommand  ?

Reply:
Done, good idea - since this base class could be used by any command

Reply:
BaseIngestionCommand is better name as we name publish/unpublish as ingest